plugins/liquid/arrow-refined-types/src/commonTest/kotlin/arrow/refinement.tests/NetworkTests.kt

```diff
@@ -1,31 +1,31 @@
 package arrow.refinement.tests
 
-import arrow.refinement.network.PrivateNetwork
-import arrow.refinement.network.Rfc1918ClassAPrivateNetwork
-import arrow.refinement.network.Rfc1918ClassBPrivateNetwork
-import arrow.refinement.network.Rfc1918ClassCPrivateNetwork
-import arrow.refinement.network.Rfc1918PrivateNetwork
-import arrow.refinement.network.Rfc2544BenchmarkNetwork
-import arrow.refinement.network.Rfc3927LocalLinkNetwork
-import arrow.refinement.network.Rfc5737Testnet1Network
-import arrow.refinement.network.Rfc5737Testnet2Network
-import arrow.refinement.network.Rfc5737Testnet3Network
-import arrow.refinement.network.Rfc5737TestnetNetwork
-import io.kotest.property.Arb
-import io.kotest.property.arbitrary.string
+// import arrow.refinement.network.PrivateNetwork
+// import arrow.refinement.network.Rfc1918ClassAPrivateNetwork
+// import arrow.refinement.network.Rfc1918ClassBPrivateNetwork
+// import arrow.refinement.network.Rfc1918ClassCPrivateNetwork
+// import arrow.refinement.network.Rfc1918PrivateNetwork
+// import arrow.refinement.network.Rfc2544BenchmarkNetwork
+// import arrow.refinement.network.Rfc3927LocalLinkNetwork
+// import arrow.refinement.network.Rfc5737Testnet1Network
+// import arrow.refinement.network.Rfc5737Testnet2Network
+// import arrow.refinement.network.Rfc5737Testnet3Network
+// import arrow.refinement.network.Rfc5737TestnetNetwork
+// import io.kotest.property.Arb
+// import io.kotest.property.arbitrary.string
 
-class NetworkTests :
-  RefinedLaws<String>(
```

Comment:
Lets tackle this in a seperate PR. We need to fix uploading the test report, so we can figure out the SEED and get a reproducible error locally for JS so we can debug.